services/movie.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+from db.models import Movie
+
+
+def get_movies(genres_ids: List[int] = None, actors_ids: List[int] = None):
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id):
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
```

Comment:
you can write `list[int]`, because we use Python 3.10

Reply:
ok ) my Python is 3.8 unfortunately. Glad to see you, Alex )

Reply:
me too)